### ISSUE TITLE
Proxy for each request in the queue?

### ISSUE THREAD

**Issue by alex-w0** [simplecrawler/simplecrawler#493]
Hi, is it possible to use an proxy for each request in the queue?

**Comment by kbychkov** [comment]
Hello! I suppose you're speaking about a [proxy rotation service](https://www.google.com/search?q=proxy+rotation+service). Simplecrawler doesn't have any out of the box but it has the [options to configure proxy](https://github.com/simplecrawler/simplecrawler#crawleruseproxy--boolean). I think you could use those options though I didn't try it myself.

**Comment by alex-w0** [comment]
Yes exactly, I wan't to scrape a large amount of data from different websites and for this purpose I need to rotate the proxy for every request. Unfortunately this option only allows me to set a proxy for each crawler.
